src/Gambit.php

```diff
@@ -76,6 +76,38 @@ public function getCampaign($id)
     }
 
     /**
+     * Call a "message" action on the Campaign endpoint.
+     *
+     * @param string $id - ID
+     * @param string $phone - Phone number, Northstar-compatible format
+     * @param string $type - The campaign message type.
+     *
+     * @see  https://github.com/DoSomething/gambit/blob/develop/documentation/endpoints/campaigns.md#send-a-campaign-message
+     *
+     * @return bool
+     */
+    public function createCampaignMessage($id, $phone, $type)
+    {
+        $payload = [
+            'phone' => $phone,
+            'type' => $type,
+        ];
+        $response = $this->post('v1/campaigns/' . $id . '/message', $payload);
+
+        if (is_null($response) || ! $this->responseSuccessful($response)) {
```

Comment:
@DFurnes @weerd, would it make sense to move this check to RestApiClient? That's the second time I'm using this logic, see `createSignup`.